rosdep/base.yaml

```
@@ -6066,6 +6066,9 @@ sdl2:
   openembedded: [libsdl2@openembedded-core]
   rhel: [SDL2-devel]
   ubuntu: [libsdl2-dev]
+sdl2-image:
+  debian: [libsdl2-image-dev]
+  ubuntu: [libsdl2-image-dev]
```

Here's the Fedora package, which also shows an entry for EPEL 7: https://src.fedoraproject.org/rpms/SDL2_image#bodhi_updates

I couldn't find anything for RHEL 8.

```suggestion
  fedora: [SDL2_Image-devel]
  rhel:
    '7': [SDL2_Image-devel]
  ubuntu: [libsdl2-image-dev]
```

---

Thanks @cottsay